blog/_posts/2022-03-30-tilt-news-march-2022.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: "tilt-news-march-2022"
+date: 2022-03-30
+author: lian
+layout: blog
+title: "Tilt News, March 2022"
+image: "/assets/images/tilt-news-march-2022/pexels-green-windmills.jpg"
+image_caption: "A dutch windmill and house all in green. Photo by Helena Jankovičová Kováčová from <a href='https://www.pexels.com/photo/a-green-windmill-near-the-house-6470648/'>Pexels</a>"
+description: "Updated posts and cool new stuff we found during spring-cleaning 🌼"
+tags:
+  - news
+---
+
+你好 Tilt 用户,
+
+This month, vacation season started at Tilt, so we’re going to have a very brief update.
+
+## Updates from the ecosystem
+Developer tooling is gaining more and more attention from engineers and investors alike. While that means tougher competition, in our opinion, this will also improve the quality of the overall ecosystem.  
```

Comment:
"Developer tooling is gaining more and more attention" feels like a weak start, because
- the category is very broad
- the phrasing lacks energy/action

maybe something like:
- The dev story on Kubernetes needs attention! More and more engineers are pitching in. VCs are taking notice.
- Kubernetes is a game-changing platform for building dev environments. Engineers and investors alike are paying attention.

?